### DISCUSSION TITLE
BAAL used in a new active learning framework for Mask R-CNN

### DISCUSSION THREAD

**Discussion by pieterblok** [baal-org/baal#176] [General]
Hello @Dref360 , @fr.branchaud-charron,

Thank you again for the nice software implementation!

For your information: I've used the BAAL software in a newly developed active learning software framework for Mask R-CNN (in Detectron2): https://github.com/pieterblok/maskal

I noticed that in your README there's a section "Papers using BaaL". I would appreciate if you can include my arXiv paper there (the paper describes the implementation of the active learning):
https://arxiv.org/abs/2112.06586

Thanks again that I was able to develop the active learning software with BAAL!

Regards,
Pieter

**Comment by Dref360** [comment]
That is awesome! Thank you for letting us know!

I will make a PR tomorrow to add your paper. Or if you prefer you can open the PR and I'll merge it.

If you have feedback on the library we would like to hear it too!

**Reply by pieterblok** [reply]
@Dref360 perfect! No rush, tomorrow is fine 😃

**Reply by Dref360** [reply]
Done!

**Reply by pieterblok** [reply]
@Dref360 that's great. Thank you very much!
